src/status_im/core.cljs

```diff
@@ -35,6 +36,7 @@
 (set! batching/fake-raf #(js/setTimeout % 0))
 (def functional-compiler (reagent.core/create-compiler {:function-components true}))
 (reagent.core/set-default-compiler! functional-compiler)
+(react-native.utils/set-convert-props-in-vectors!)
```

Comment:
it's better to do not have this function in utils at all, just move code here `(set! template/convert-prop-value convert-prop-value)`

Reply:
btw im not sure why is it in `react-native.utils` it should be in `utils/somethin`